### PR TITLE
Use oc-nav-bar to implement Top-Bar

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Screenshot from 2019-04-11 16-22-46

### CODE REVIEW

**Comment by felixheidecke** [comment]
I suggest dropping sticky for now. The "outer shell" around the app including the topbar should remain @ a fixed 100vh where as app should scroll. So Topbar will remain stationary and "sticky" can and should be used inside the apps only.

File: src/components/Top-Bar.vue [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-    <oc-topbar-logo icon="menu" title="Files" slot="left" @click="toggleSidebar(!isSidebarVisible)"></oc-topbar-logo>
-    <oc-topbar-item slot="title">
-      <oc-img :src="configuration.theme.logo.big" style="height: 60px" />
-    </oc-topbar-item>
-    <oc-topbar-item slot="right">
-      <Avatar class="uk-margin-small-right" :userid="user.id" />
-      <span>{{ user.displayname }}</span>
-    </oc-topbar-item>
-  </oc-topbar>
+  <oc-navbar tag="header" class="oc-topbar" :sticky="true" :offset="0">
```
Comment (felixheidecke) [inline src/components/Top-Bar.vue]: Let's drop sticky here as it's not intended to be used with the topbar

Comment (DeepDiver1975) [reply src/components/Top-Bar.vue]: > I suggest dropping sticky for now. The "outer shell" around the app including the topbar should remain @ a fixed 100vh where as app should scroll. So Topbar will remain stationary and "sticky" can and should be used inside the apps only.

I assume this requires some scss magic - if so - please open an issue in ods - THX

Comment (DeepDiver1975) [reply src/components/Top-Bar.vue]: stiicky killed for now

**Pull request merged**